content/How an engineer can be your best partner.md

```diff
@@ -0,0 +1,86 @@
+
+Have you ever said one of these tricky phrases?
+  - I'm sure this feature is going to rock it!
+  - We need this feature before going live, it is a must!
+  - Users will use this for sure!
+
+Well, if you found yourself saying any of those without any numbers that support you, you may probably be making a lot of assumptions.
+So is like you are playing roulette, due to lacking data to prove your hypothesis.
+Give growth a try! And find out how to move smarter with your development team along with really learning what your users want!
+
+## 1) What's GROWTH
+
+I see growth as a way of working around data analysis based decisions. So first let's explain a bit what data analysis is:
+
+`Data analysis is defined as a process of cleaning, transforming, and modeling data to discover useful information for business decision-making. The purpose of Data Analysis is to extract useful information from data and taking the decision based upon the data analysis`
+
+You may see that `data` word came up a lot, that's not because I want to be redundant, instead is because I want to emphasize it as a really important concept. Nowadays the more data you have the more power you have to make better decisions.
+
+
+## 2) How to start?
+
+As I mentioned before, start gathering data, it doesn't need to be super high-quality data, you will start learning from your users and your business reality and improve step by step.
+A simple step you can take at a really low cost is adding some forms and analytics to your landing page. This way you will start getting feedback from users and also start knowing your users interactions and interests.
+
+First, see an initial approach on how you should approach your changes:
+![hypothesis process](images/build-measure-learn.jpg)
+
+You will start building something as small as possible (to lessen the number of mistakes you can make).
+Then you measure it (please! test it with users, they are the only ones with the truth on this).
+Finally, learn from your measures and keep building, but now with a clearer idea of what users want.
+
+To plan what to build you can follow this simple process, also to manage each specific change.
+![hypothesis process](images/hypothesis.jpg)
+
+  - Write down your hypothesis based on your business goals
+  - Plan tests on your product to validate your hypothesis
+  - Measure
+  - Verify your numbers against what you thought
+
+The most important step at the end is to make decisions based on your learnings, adjusting your upcoming features/hypothesis based on learnings.
+
+## 3) Common mistakes
+
+#### First and most usual mistake: assume you know how users will behave
+
+You will probably be wrong, instead of that write a hypothesis and try to prove it. It can be as simple as "I want to prove that I get more sales if I place the BUY button in some place with some color" or more complex like "I think users are tempted to buy if I offer discounts after 10 minutes looking items in my store"
```

Comment:
```suggestion
You will probably be wrong, instead of that write a hypothesis and try to prove it. It can be as simple as "I want to prove that I get more sales if I place the BUY button at the top with a brighter color" or more complex like "I think users are tempted to buy if I offer discounts after 10 minutes of looking at items in my store"
```